includes/CAPx/Drupal/Entities/CFEntityType.php

```diff
@@ -4,9 +4,10 @@
  * @author
  */
 
-namespace CAPx\Drupal\Entities;
```

Comment:
In short, because namespacing and autoloading sucks in Drupal 7.

The entity system and the way you hook into it prevented me from keeping the namespace in D7 as the path can no longer be absolute in the entity info definition. After that kerfuffle, I had to shift all the uses of the entity so that they matched the interface that Drupal 7 uses. It was a pain to just weed out the ones that had to be changed in order for this to work and to keep the others in hopes that I shift less code and introduce fewer errors.

Reply:
sounds about right for D7

Reply:
Ah woot. thanks. GTG then?